actioncable/test/client_test.rb

```diff
@@ -12,24 +12,15 @@ class ClientTest < ActionCable::TestCase
   WAIT_WHEN_NOT_EXPECTING_EVENT = 0.2
 
   def setup
-    # TODO: ActionCable requires a *lot* of setup at the moment...
-    ::Object.const_set(:ApplicationCable, Module.new)
-    ::ApplicationCable.const_set(:Connection, Class.new(ActionCable::Connection::Base))
-
-    ::Object.const_set(:Rails, Module.new)
-    ::Rails.singleton_class.send(:define_method, :root) { Pathname.new(__dir__) }
-
     ActionCable.instance_variable_set(:@server, nil)
     server = ActionCable.server
-    server.config = ActionCable::Server::Configuration.new
-    inner_logger = Logger.new(StringIO.new).tap { |l| l.level = Logger::UNKNOWN }
-    server.config.logger = ActionCable::Connection::TaggedLoggerProxy.new(inner_logger, tags: [])
+    server.config.logger = Logger.new(StringIO.new).tap { |l| l.level = Logger::UNKNOWN }
```

Comment:
Couldn't remove this without seeing circular require warnings throw in the Active Record connection management concern. Haven't yet figured out why that is.